content/developer/reference/backend/orm.rst

```diff
@@ -1229,16 +1229,18 @@ essentially extending it in-place. This is useful to add new fields or methods
 to existing models (created in other modules), or to customize or reconfigure
 them (e.g. to change their default sort order)::
```

Comment:
`::` makes what follow a code-block, if you change the thing for a `.. code-block` make sure to remove `::`
```suggestion
them (e.g. to change their default sort order)
```